### ISSUE TITLE
[Bug] A job that runs `cargo test -- --include-ignored` fails to work 

### ISSUE THREAD

**Issue by rszyma** [Canop/bacon#130]
I've defined a custom job called `test-all`:
```toml
[jobs.test-all]
command = ["cargo", "test", "--color", "always", "--", "--color", "always", "--include-ignored"]
need_stdout = true
```
The difference between this and default `test` job is that it uses `--include-ignored` flag to also run tests with `#[ignore]` attribute.

When I run this job and any of the tests fail this is the result:
![image](https://user-images.githubusercontent.com/44318430/231746548-134f0863-5e1b-473e-b15d-4c294cf40ba6.png)

There are 2 problems here:
1. I can't scroll the output.
2. It fails with `Command error code: 101`. I've skimmed through the code and figured out that `AppState.cmd_result` is `CommandResult::Failure`, so I the command output failed to parse maybe?

**Comment by rszyma** [comment]
Actually, this is not a problem with `--include-ignored` flag, but with bacon having problems parsing color codes when running `cargo test -- --color always`. 

Because of color codes in almost every line, this branch is not taken when it should:

https://github.com/Canop/bacon/blob/2f6ad15cecd3312a2b836424462c9ec2a80d46e7/src/line_analysis.rs#L28

Was there any reason why the color escape sequences are not stripped from output in the first place, before even attempting to parse it?

Related: https://github.com/Canop/bacon/issues/124#issuecomment-1483761611

**Comment by rszyma** [comment]
@Canop

**Comment by Canop** [comment]
> Was there any reason why the color escape sequences are not stripped from the command output in the first place, before even attempting to parse it?

The idea was that by stripping style information you're more likely to misinterpret the output. 

Related: https://github.com/Canop/bacon/pull/140

**Comment by Canop** [comment]
This should be fixed with version 2.9.0.

Please reopen if it isn't.
